clr-k8s-examples/create_stack.sh

```diff
@@ -89,14 +99,19 @@ function kata() {
 
 function cni() {
 	# note version is not semver
-	CANAL_VER=${1:-v3.3}
-	CANAL_URL="https://docs.projectcalico.org/$CANAL_VER/getting-started/kubernetes/installation/hosted/canal"
+	CANAL_VER=${1:-v3.9}
+	CANAL_URL="https://docs.projectcalico.org/${CANAL_VER}/manifests"
+	if [[ "$CANAL_VER" == "v3.3" ]]; then
+		CANAL_URL="https://docs.projectcalico.org/v3.3/getting-started/kubernetes/installation/hosted/canal"
+	fi
 	CANAL_DIR="0-canal"
 
 	# canal manifests are not kept in repo but in docs site so use curl
 	mkdir -p "${CANAL_DIR}/overlays/${CANAL_VER}/canal"
 	curl -o "${CANAL_DIR}/overlays/${CANAL_VER}/canal/canal.yaml" "$CANAL_URL/canal.yaml"
-	curl -o "${CANAL_DIR}/overlays/${CANAL_VER}/canal/rbac.yaml" "$CANAL_URL/rbac.yaml"
+	if [[ "$CANAL_VER" == "v3.3" ]]; then
+  		curl -o "${CANAL_DIR}/overlays/${CANAL_VER}/canal/rbac.yaml" "$CANAL_URL/rbac.yaml"
```

Comment:
I assume if we don't have 3.3 then we don't need the rbac yaml?

Reply:
That's correct.